src/parsing/tgraph.py

```diff
@@ -357,3 +362,34 @@ async def telegraph_ify(self):
                     f'Network error ({type(e).__name__}) occurred when creating telegraph page, will retry')
                 return await self.telegraph_ify()
             raise e
+
+
+def telegraph_file_upload(file_url):
+    '''
+    Sends a file to telegra.ph storage and returns its url
+    Works ONLY with 'gif', 'jpeg', 'jpg', 'png', 'mp4'
+
+    Parameters
+    ---------------
+    path_to_file -> str, path to a local file
+
+    Return
+    ---------------
+    telegraph_url -> str, url of the file uploaded
+
+    >>>telegraph_file_upload('test_image.jpg')
+    https://telegra.ph/file/16016bafcf4eca0ce3e2b.jpg
+    >>>telegraph_file_upload('untitled.txt')
+    error, txt-file can not be processed
+    '''
+    response = requests.get(file_url)
```

Comment:
Use the `web.get` wrapper (in `src/web/req.py`). Or else proxy and other features will not be applied.